shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.mm

```diff
@@ -130,40 +123,37 @@ - (void)onDisplayLink:(CADisplayLink*)link {
 
   recorder->RecordVsync(frame_start_time, frame_target_time);
   if (_allowPauseAfterVsync) {
-    display_link_.get().paused = YES;
+    link.paused = YES;
   }
   callback_(std::move(recorder));
 }
 
 - (void)invalidate {
-  [display_link_.get() invalidate];
+  [_displayLink invalidate];
+  _displayLink = nil;  // Break retain cycle.
```

Comment:
The test I wrote to validate the display link is released when `VSyncClient` invalidates and deallocs doesn't pass without this line (and also doesn't pass on main with the current code).